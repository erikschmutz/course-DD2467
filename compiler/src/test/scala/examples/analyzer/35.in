let p = [|1, 2|];;
p(1);;

(*====================
val p : int[||]
- :int
====================*)