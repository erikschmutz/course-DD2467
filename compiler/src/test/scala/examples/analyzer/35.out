AnalyzerResult(
    Integer(),
    Enviroment(
        List(
            ValEntry(
                Identifier(p),
                ArrayList(Integer())
            )
        )
    )
)